.github/workflows/golangci-lint.yml

```diff
@@ -28,12 +28,12 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@f111f3307d8850f501ac008e886eec1fd1932a34 # v5.3.0
         with:
-          go-version: 1.23.x
+          go-version: 1.24.x
       - name: Install snmp_exporter/generator dependencies
         run: sudo apt-get update && sudo apt-get -y install libsnmp-dev
         if: github.repository == 'prometheus/snmp_exporter'
       - name: Lint
         uses: golangci/golangci-lint-action@ec5d18412c0aeab7936cb16880d708ba2a64e1ae # v6.2.0
         with:
           args: --verbose
-          version: v1.63.4
+          version: v1.64.5
```

Comment:
```suggestion
          version: v1.63.4
```

Reply:
Remove to golangci-lint version v1.63.4
Update golangci-lint version by Next PR